hack/jenkins/job-configs/kubernetes-build.yaml

```diff
@@ -6,7 +6,7 @@
         numToKeep: 200
     builders:
         - shell: 'bash <(curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-started.sh")'
-        - shell: './hack/jenkins/build.sh'
+        - shell: 'timeout -k {kill-timeout}m 30m ./hack/jenkins/build.sh'
```

Comment:
it's not strictly necessary, but I wonder if this would read clearer to have the signal-timeout also specified as a variable.